src/components/Note/Note.tsx

```diff
@@ -120,10 +119,14 @@ export const Note = (props: NoteProps) => {
             />
           </div>
         ) : (
-          <main className={classNames("note__text", {"note__text--extended": !showNoteReactions})}>
-            <NoteTextContent text={note.text} truncate />
+          <main className={classNames("note__container")}>
+            <div className={classNames("note__text", {"note__text--extended": !showNoteReactions})}>
+              <NoteTextContent text={note.text} truncate />
+            </div>
+            {note.edited && <div className={classNames("note__marker-edited")}>({t("Note.edited")})</div>}
```

Comment:
don't need to use the `classNames` package here if only one class is provided. At least for my taste, it isn't wrong ofc